langchain/prompts/base.py

```diff
@@ -5,12 +5,20 @@
 from typing import Any, Dict, List, Optional, Union
 
 import yaml
+from jinja2 import Template
 from pydantic import BaseModel, Extra, root_validator
 
 from langchain.formatting import formatter
 
+
+def jinjaFormatter(template: str, **kwargs: Any) -> str:
```

Comment:
nit: we follow google style for naming conventions, so should be jinja_formatter

https://google.github.io/styleguide/pyguide.html#316-naming

Reply:
Yes, you are on point with this. I prefered to discuss with you upfront before fixing all theses things.
I'll take care of them today !